src/Symfony/Component/DependencyInjection/Tests/Compiler/AutowirePassTest.php

```diff
@@ -700,7 +697,14 @@ public function testSetterInjectionCollisionThrowsException()
         (new AutowireRequiredMethodsPass())->process($container);
 
         $pass = new AutowirePass();
-        $pass->process($container);
+
+        try {
+            $pass->process($container);
+        } catch (AutowiringFailedException $e) {
+        }
+
+        $this->assertNotNull($e);
+        $this->assertSame('Cannot autowire service "setter_injection_collision": argument "$collision" of method "Symfony\Component\DependencyInjection\Tests\Compiler\SetterInjectionCollision::setMultipleInstancesForOneArg()" references interface "Symfony\Component\DependencyInjection\Tests\Compiler\CollisionInterface" but no such service exists. You should maybe alias this interface to one of these existing services: "c1", "c2".', $e->getMessage());
```

Comment:
What's the difference with the previous way this test was written?
This one doesn't tel about `Did you create...`

Reply:
`@expectedExceptionMessage` uses `assertContains` IIRC